### PR TITLE
Added Commas to income totals and decimal changes

### LINKED ISSUE #277: Add commas to income totals
The total income for a given income type is displayed on each income details page.  This value is formatted as a number with a preceding **$**

Including commas appropriately would make the value easier to read. e.g. $3,125 vs $3125

### PR DESCRIPTION
**Pull request by gangaramani-nikhil**
fixes #277

### CODE REVIEW

**Comment by trevorshannon** [comment]
(not sure why the Netlify checks are failing...everything looks ok from here)

Thanks!  Looks good.

I like that no decimals are rendered if none are entered. I also just noticed that the rendering of decimals ending in zero could be improved.  Note that this is _not_ a newly-introduced behavior. The current version of the site also does the decimal truncation.  So, if you feel this additional improvement is outside the scope of the original issue, no problem.  I can write a new issue.

Screenshot 2023-03-26 at 7 28 43 PM

I know it's easy to always show decimals or never show decimals.  Not sure about switching between the two.

**Comment by gangaramani-nikhil** [comment]
Hi @trevorshannon ,
I can work on this same issue to solve the decimals ending in zero problem.
Just to be sure:
Input --------------> expected output
100                                100
100.5                             100.50
100.50                           100.50
100.235                         100.23


Is this the expected behaviour ?

do let me know Thanks

**Comment by gangaramani-nikhil** [comment]
I think it's not allowing me to create another pull request as I already have 1 pull request open.

**Comment by gangaramani-nikhil** [comment]
Hi @trevorshannon ,
I have made the changes
| Input | Expected Output | screenshot|
|--------|--------|--------|
| 100 | 100 | 
image|
| 100.5 | 100.50 |
image|
| 100.50 | 100.50 |
image|
| 100.235 | 100.23 | 
image|
 |100.00 | 100 |
image|

**Comment by gangaramani-nikhil** [comment]
Sure will create new branch in my fork

File: src/site/_includes/js/eligibility.js [hunk 1]
```diff
@@ -457,7 +457,11 @@ function updateIncomeTotal() {
   for (input of inputs) {
     sum += Number(input.value);
   }
-  totalDisplay.textContent = sum;
+  sum = parseFloat(sum).toFixed(2)
```
Comment (trevorshannon) [inline src/site/_includes/js/eligibility.js:460]: `sum` should already be a `Number` thanks to the conversion done in the summation loop.  So I think `sum = sum.toFixed(2);` should work ok.

**Review by trevorshannon — CHANGES_REQUESTED** [review]
Thank you!  Just some minor comments.

**Comment by gangaramani-nikhil** [comment]
I have made the changes as requested

File: src/site/_includes/js/eligibility.js [hunk 2]
```diff
@@ -457,7 +457,12 @@ function updateIncomeTotal() {
   for (input of inputs) {
     sum += Number(input.value);
   }
-  totalDisplay.textContent = sum;
+  sum = sum.toFixed(2)
```
Comment (trevorshannon) [inline src/site/_includes/js/eligibility.js:460]: Just one missing semicolon

**Review by trevorshannon — COMMENTED** [review]
Looks good!  After the semicolon addition, we should be ready to merge.

File: src/site/_includes/js/eligibility.js (hunk 2 above)
Comment (gangaramani-nikhil) [reply src/site/_includes/js/eligibility.js]: done

**Review by trevorshannon — APPROVED** [review]

**Pull request merged**